File: chroma-league/src/main/java/com/bonepl/chromaleague/hud/animations/EnemyCloudDragonKillAnimation.java
package com.bonepl.chromaleague.hud.animations;

import com.bonepl.chromaleague.rest.eventdata.DragonType;

import static com.bonepl.chromaleague.hud.PredefinedKeySets.BLACKWIDOW_FUNCTIONAL;

public class EnemyCloudDragonKillAnimation extends StaticPartialBlinkingAnimation {
    public EnemyCloudDragonKillAnimation() {
        super(BLACKWIDOW_FUNCTIONAL, 8, DragonType.CLOUD.getColor());
    }
}

File: src/hud/animations/EnemyCloudDragonKillAnimation.ts
import {DragonType} from '../../rest/eventdata/DragonType.js';
import {EnemyDragonKillBlinkingAnimation} from './EnemyDragonKillBlinkingAnimation.js';

export class EnemyCloudDragonKillAnimation extends EnemyDragonKillBlinkingAnimation {
    constructor() {
        super(DragonType.CLOUD);
    }
}
